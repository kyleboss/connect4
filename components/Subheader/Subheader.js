import styled from "styled-components";
import { TABLET_AND_LARGER_SIZE } from "@/utils/constants";

const Subheader = styled.h2`
  font-size: 1rem;
  margin-bottom: 30px;
  font-weight: 800;
  font-style: italic;
  text-align: center;

  @media (min-width: ${TABLET_AND_LARGER_SIZE}) {
    font-size: 2rem;
    margin-bottom: 50px;
  }
`;

export default Subheader;
