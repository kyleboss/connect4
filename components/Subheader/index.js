export * from './Subheader';
export { default } from './Subheader';
