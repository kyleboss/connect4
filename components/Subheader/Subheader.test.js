import React from "react";
import { render } from "@testing-library/react";

import Subheader from "./Subheader";

describe("Subheader", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Subheader>Sample Text</Subheader>);
    expect(asFragment()).toMatchSnapshot();
  });
});
