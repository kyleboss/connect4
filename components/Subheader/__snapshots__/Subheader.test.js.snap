// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subheader renders correctly 1`] = `
<DocumentFragment>
  <h2
    class="sc-773df940-0 icsDSx"
  >
    Sample Text
  </h2>
</DocumentFragment>
`;
